cypress/support/component-helper/common-steps.js

```diff
@@ -23,11 +23,28 @@ export const parseToIntElement = (elem) => {
   return parseInt(inputString);
 };
 
-export const checkGoldenOutline = (elem, outlineWidthPx = 3) => {
-  const outlineWidth = parseToIntElement(elem.css("outline-width"));
-  expect(elem.css("outline-color")).to.equals("rgb(255, 181, 0)");
-  expect(elem.css("outline-style")).to.equals("solid");
-  expect(outlineWidth).to.be.within(outlineWidthPx - 1, outlineWidth + 1);
+export const checkGoldenOutline = (
```

Comment:
If this is no longer only checking `outline` maybe something more generic would be better wrt to naming of the function and args:

```
  export const checkFocusStyling = (elem, focusStyleWidthPx = 3, focustStyleProperty) => ...
```

Reply:
this function actually checks only the focus (golden outline) so I would leave it as it is